lib/shelley/bench/Restore.hs

```diff
@@ -178,98 +192,126 @@ cardanoRestoreBench tr c socketFile = do
 
     prepareNode networkProxy socketFile np vData
     runBenchmarks
-        [ bench ("restore " <> network <> " seq")
-            (bench_restoration @_ @ShelleyKey
-                networkProxy
-                tr
-                socketFile
-                np
-                vData
-                "seq.timelog"
-                (walletSeq networkProxy))
-
-        , bench ("restore " <> network <> " rnd")
-            (bench_restoration @_ @ByronKey
-                networkProxy
-                tr
-                socketFile
-                np
-                vData
-                "rnd.timelog"
-                (walletRnd mkRndState))
-
-        , bench ("restore " <> network <> " 1% rnd")
-            (bench_restoration @_ @ByronKey
-                networkProxy
-                tr
-                socketFile
-                np
-                vData
-                "1-percent-rnd.timelog"
-                (walletRnd $ mkRndAnyState @1))
-
-        , bench ("restore " <> network <> " 0.1% any")
-             (bench_restoration @_ @ShelleyKey
-                 networkProxy
-                 tr
-                 socketFile
-                 np
-                 vData
-                 "0.1-percent-any.timelog"
-                 (initAnyState "Benchmark 0.1% Wallet" 0.001))
-
-        , bench ("restore " <> network <> " 0.5% any")
-             (bench_restoration @_ @ShelleyKey
-                 networkProxy
-                 tr
-                 socketFile
-                 np
-                 vData
-                 "0.5-percent-any.timelog"
-                 (initAnyState "Benchmark 0.5% Wallet" 0.005))
-
-        , bench ("restore " <> network <> " 1% any")
-             (bench_restoration @_ @ShelleyKey
-                 networkProxy
-                 tr
-                 socketFile
-                 np
-                 vData
-                 "1-percent-any.timelog"
-                 (initAnyState "Benchmark 1% Wallet" 0.01))
+        [ bench_restoration @_ @ShelleyKey
+            networkProxy
+            tr
+            socketFile
+            np
+            vData
+            "seq.timelog"
+            (walletSeq "Seq Empty Wallet" $ mkSeqState networkProxy)
```

Comment:
_Could_ call it `Seq 0% wallet` for consistency. Or perhaps what you have makes sense given that the Empty tests use the fully real states instead of the wrapped `AnyState @p` ones.

Reply:
0% sounds like a fine name :)